lib/DDG/Meta/Data.pm

```diff
@@ -50,8 +50,8 @@ unless(%ia_metadata){
     unless($ENV{NO_METADATA_DOWNLOAD}){
         my $ua = LWP::UserAgent->new;
         $ua->timeout(5);
-        $ua->default_header('Accept-Encoding' => scalar HTTP::Message::decodable());
-        my $res = $ua->mirror('http://ddg-community.s3.amazonaws.com/metadata/repo_all.json.bz2', $f);
+        $ua->default_header('Accept-Encoding' => 'gzip');
+        my $res = $ua->mirror('https://duckduckgo.com/iameta.json', $f);
```

Comment:
There's a pretty basic race condition here in that some nodes may not have the new data by the time this is invoked.

Wonder if it should source from 127.1?